pybaseball/analysis/string_similarity.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+
+def levenshtein(seq1: str, seq2: str) -> np.int64:
```

Comment:
I think it'd be better to use a library such as [fuzzywuzzy](https://github.com/seatgeek/fuzzywuzzy) instead of writing a custom levenshtein function. is there a reason to use the custom version here?

Reply:
I guess that's up to preference, I figured adding another dependency for a ~20 line function would be overkill but if we'd rather use an external library, that's fine with me.

Reply:
This looks like a potential perf-killer, do you have measurements on this?